vorestation.dme

```diff
@@ -2618,17 +2629,6 @@
 #include "code\modules\rogueminer_vr\wrappers.dm"
 #include "code\modules\rogueminer_vr\zone_console.dm"
 #include "code\modules\rogueminer_vr\zonemaster.dm"
-#include "code\modules\research\designs\ai_modules.dm"
-#include "code\modules\research\designs\circuit_assembly.dm"
-#include "code\modules\research\designs\circuits.dm"
-#include "code\modules\research\designs\illegal.dm"

```

Comment:
Ohm, what is this

Reply:
Oh, I forgot to unstage this. It's something that popped up like... a week ago, when I did a fetch/merge.